.env.github

```diff
@@ -22,4 +22,7 @@ EVOLUTION_API_ENDPOINT_TOKEN=EVOLUTION_API_ENDPOINT_TOKEN
 
 # Enable user register page
 ENABLE_USER_SIGNUP=true
-OPENAI_API_KEY='test-env-key'
\ No newline at end of file
+OPENAI_API_KEY='test-env-key'
+
+#language available: pt-BR, es, en
+LANGUAGE=en
```

Comment:
This line adds the LANGUAGE environment variable to set the system’s default translation. This is useful because if the translation ever needs to be changed, the application won’t have to be redeployed.